src/main/resources/db/changelog/changes/create-cars-table.yml

```diff
@@ -0,0 +1,47 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-cars-table
+      author: Mykola
+      changes:
+        - createTable:
+            tableName: cars
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: model
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: brand
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: type
+                  type: enum('SEDAN', 'SUV', 'HATCHBACK', 'UNIVERSAL')
+                  constraints:
+                    nullable: false
```

Comment:
so how we doo???? with varchar(255) or with enums?? )

Reply:
I think varchar is better to go we do not need any restrictions in demands